common/components/WalletDecrypt/components/DeterministicWalletsModal.tsx

```diff
@@ -113,21 +122,20 @@ class DeterministicWalletsModalClass extends React.Component<Props, State> {
         handleClose={onCancel}
       >
         <div className="DWModal">
+          {/* TODO: replace styles for flexbox with flexbox classes in https://github.com/MyEtherWallet/MyEtherWallet/pull/850/files#diff-2150778b9391533fec7b8afd060c7672 */}
```

Comment:
From my comments over there, I think I would prefer flex ui components over classes. Just a thought.

Reply:
@james-prado Do you mind taking a look here?

Reply:
For flexbox styles I prefer using classes. Elements wrapped in a `display: flex` style tend to have a lot of custom styles specific to each element that is hard to replicate with components, and all of this is becomes even harder when trying to provide support for phone and tablet screen widths 
ie. placing spacer components (empty divs with `flex-grow: 1`), wrapping, aligning

Reply:
Also, I'm not big on using many custom classes for flex-box, but a few classes that are regularly used and aren't too specific seem to work really well with the vast amount of configuration options for flexbox